scripts/zkai_d128_block_receipt_composition_gate.py

```
@@ -0,0 +1,1020 @@
+        "slice chain commitment",
+    )
+    expect_equal(
+        payload.get("evidence_manifest_commitment"),
+        blake2b_commitment(payload["source_evidence_manifest"], EVIDENCE_MANIFEST_DOMAIN),
+        "evidence manifest commitment",
+    )
+    _validate_source_hashes(payload)
+    _validate_receipt(payload)
+    summary = payload.get("summary")
+    if not isinstance(summary, dict):
+        raise D128BlockReceiptError("summary must be an object")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)